aten/src/ATen/native/mps/operations/View.mm

```diff
@@ -549,12 +555,12 @@ static IntArrayRef updateTensorBaseShape(const Tensor& self)
 //            |    /          \   |
 //            |   /            \  |
 //            NonView T         NonView T
-static ViewCachedGraph* createViewGraph(const Tensor& self, IntArrayRef size, IntArrayRef stride, int64_t storage_offset, bool needsScatter)
+static ViewCachedGraph* createViewGraph(const Tensor& self, IntArrayRef size, IntArrayRef stride, int64_t storage_offset, bool needsScatter, const Tensor &other)
```

Comment:
- Two different names "updates" and "other" for the same tensor is confusing when reading the code. Can we use one name only?
- Also can you move `const Tensor &other` to second arg beside self tensor? (I remember someone complained about a similar cosmetic issue when we were upstreaming).

Reply:
> Two different names "updates" and "other" for the same tensor is confusing when reading the code. Can we use one name only?

In case of `gather` there is no `updates` Tensor and it was confusing to call it like that for both scatter/gather. I'll update it to `other` for all the places and put a comment that in case of `scatter` it's representing the updates tensor.